packages/components/src/components/Input/Input.tsx

```diff
@@ -25,6 +25,10 @@ export type InputProps = {
      * A boolean value indicating whether or not the input component should span across multiple lines of text or wrapped to one line.
      */
     multiline?: boolean;
+    /**
+     * A boolean value indicating whether or not the input component should span across multiple lines of text or wrapped to one line.
+     */
+    allowCancel?: boolean;
```

Comment:
Should we remove the `allowCancel` prop from this component? 
I guess it is no longer used since we moved the cancel funcitonality to the search bar instead